include/openssl/ssl.h

```diff
@@ -282,6 +282,9 @@ typedef int (*SSL_custom_ext_parse_cb_ex) (SSL *s, unsigned int ext_type,
 /* Typedef for verification callback */
 typedef int (*SSL_verify_cb)(int preverify_ok, X509_STORE_CTX *x509_ctx);
 
+/* In TLSv1.3 allow a non-(ec)dhe based kex_mode */
+# define SSL_OP_ALLOW_NO_DHE_KEX                         0x00000001U
```

Comment:
After thinking about it for a while, I was going to say something similar to what @kaduk stated. All those option bits in SSL_OP_ALL that were removed in 1.1.0f were removed from OpenSSL in 1.1.0, so it's "safe" as far as compatibility is concerned. However, since SSL_OP_ALL was not changed in 1.1.0 until 1.1.0f, none of the 0x00000BFF bits should be re-used in 1.1.x. Once 1.2 is introduced, those bits become available.